health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MapView.js

```
@@ -74,7 +88,11 @@ const MapView = ({ fileData }) => {
     }
   }, [geoJsonData]);
 
-  return <div id="map" style={{ height: "100vh", width: "100%" }}></div>;
+  return (
+    <div>
+      <div id="map" style={{ height: "100vh", width: "100%" , display: "inline-block" }}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)